friends/src/components/FriendForm.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class FriendForm extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      newFriend: {
+        name: '',
+        age: '',
+        email: '',
+      },
+    }
+  }
+
+  handleInputChange = (event) => {
+    event.persist();
+    const value = event.target.value;
+
+    this.setState(prevState => ({
+      newFriend: { ...prevState.newFriend, [event.target.name]: value }
+    }));
+  }
+
+  handleSubmit = event => {
+    event.preventDefault();
+
+    axios.post("http://localhost:5000/friends", this.state.newFriend)
+      .then(response => {
+        this.props.updateFriends(response.data);
+
+      })
+      .then(() =>{
+        this.setState({newFriend: {
+          name: '',
+          age: '',
+          email: '',
+        }})
+      })
+      .catch(err => console.log(err));
+    // need to prevent adding empty friend
+  };
+
+  render () {
+    return (
+     <div className="friend-form">
+       <h2>Add a new friend</h2>
+       <form onSubmit={this.handleSubmit}>
+         <label> Name:
+         <input
+           type="text"
+           name="name"
+           onChange={this.handleInputChange}
+           placeholder="Enter name"
+           value={this.state.newFriend.name}
+         />
+         </label>
+         <div className="separator-line" />
```

Comment:
What's going on with this? I know styling isn't the focus of this project, but there are better ways to put a line between your inputs. What about flexbox? what about display block? Adding empty divs to the dom isn't a very good solution to this problem.